### ISSUE TITLE
Creating dedicated server through client crashes or breaks rendering

### ISSUE THREAD

**Issue by thedt** [JACoders/OpenJK#217]
Openjk seems to always crash when hosting a server using openjk.x86.exe ,but on openjkded.x86.exe it does fine.

**Comment by xycaleth** [comment]
Update:

Currently, if you try to create a dedicated server through the client (Play -> Create Game), rendering becomes completely broken. This is related in some way to the modular renderer. Screenshot below:

![Bad renderer](https://dl.dropboxusercontent.com/u/874909/jka/shot2015-02-07_17-03-56.jpg)

**Comment by mrwonko** [comment]
Does this still happen?

**Comment by ensiform** [comment]
This won't be fixed for the immediate future.

**Comment by Razish** [comment]
Sure it will, if someone provides a patch. It's just not a priority for us.
Creating a *solo game* seems to work fine, but not creating a dedicated server through the client executable.

**Comment by ensiform** [comment]
My fixes depend on getting the frame / sleep code fixed first.  Plus it also semi-ties in to having the working game restarts.  But its not finished either.

**Comment by ensiform** [comment]
I think we could remove your fix and opt for a better workaround in SV_SpawnServer though.

Forcefully set dedicated to 0 if not DEDICATED bin. And/Or also ERR_DROP to menu saying its currently unsupported?

**Comment by CountGlacier** [comment]
HI 

1.) I just need help with openjk.x86 when i select a normal server it loads then crashes.
2) How do i enable mods in openjk.x86 because i see there isn't a mods tab in settings or does it run automatically when i log onto a server with a specific mod? (I have the mod installed.)

(i am new to this coding thing and modding thing, please excuse the amateurish questions.)
Thanks

**Comment by Razish** [comment]
@CountGlacier By 'normal server' do you mean `Solo Game`? This should work. Make sure it's not related to #751 
The mods menu still exists and functions as it always did.

**Comment by CountGlacier** [comment]
@Razish No not solo game, multiplayer servers when i connect to a multiplayer server it loads then crashes. (#751) I did look at that Issue but i run openjk application. plus the +set fs_game OpenJK  i have already entered on steam and i am able to get onto multiplayer but it only crashes when i log onto a server. i don't have that problem. (ok problem solved) i got multiplayer working so now it does not crash when i log onto a server. But now how do i enable the mod Movie Battles 2. i took a chance and log onto a server with the mod to my surprise i was loaded into the mod but the rendering was bad and the text was gone. I just need to be told/show how to enable movie battles 2 inside openjk (because it is only a multiplayer mod) I hope you understand what i am saying here. 

Thanks

**Comment by ensiform** [comment]
MB2 requires MB2's special version I thought. starting with +set fs_game MBII or whatever the MB2 mod folder is called instead of OpenJK is worth a shot.

**Comment by CountGlacier** [comment]
@ensiform yeah i will try what you said. thanks and I never said i had problems with openjk.x86

**Comment by ensiform** [comment]
You did though.  +set fs_game OpenJK is irrelevant to wanting to run a mod like MB2.

**Comment by CountGlacier** [comment]
I Think i know what the problem is i will get back to you if it works. if not well.....
